src/metadata/entity/index.ts

```diff
@@ -20,7 +20,13 @@ export class IasqlDatabase {
   region: string;
 
   @Column()
-  pgUser: string
+  pgUser: string;
+
+  @Column({
+    // default to true for databases that already exists
+    default: true,
+  })
+  isReady: boolean;
```

Comment:
would it be possible to know if a database was created with direct connect? perhaps don't default to `true` so that we can write a basic query where isReady is null that tells us how often each connection flow is used?